pkg/utils/utils.go

```diff
@@ -315,7 +318,7 @@ func GetAppsubs(m *operatorsv1.MultiClusterHub) []types.NamespacedName {
 		{Name: "volsync-addon-controller-sub", Namespace: m.Namespace},
 	}
 	if m.Spec.EnableClusterBackup {
-		appsubs = append(appsubs, types.NamespacedName{Name: "cluster-backup-chart-sub", Namespace: m.Namespace})
+		appsubs = append(appsubs, types.NamespacedName{Name: "cluster-backup-chart-sub", Namespace: "open-cluster-management-backup"})
```

Comment:
```suggestion
		appsubs = append(appsubs, types.NamespacedName{Name: "cluster-backup-chart-sub", Namespace: ClusterSubscriptionNamespace})
```
Since you have the variable in this package and used it in other places it would be consistent to use it here rather than a hardcoded string

Reply:
Updated!